charts/newrelic-prometheus-agent/values.yaml

```diff
@@ -182,8 +182,21 @@ config:
   # -- It allows defining scrape jobs for Kubernetes in a simple way.
   # @default -- See `values.yaml`
   kubernetes:
+
+    # By default, all targets discovered are scraped.
+    # If curated_experience is enabled or any regex is specified,
+    # then merely the targets having one of the specified labels matching those regexes or the ones of the curated experiences are scraped.
+    # Notice that this configuration option applies to all configured kubernetes jobs.
+    # Under the hood relabel_configs with action 'keep' are generated, consider it in case any custom extra_relabel_config is needed
+    curated_experience:
+      enabled: false
+      source_labels: ["app.kubernetes.io/name", "k8s-app", "app"]
+      additional_regexes: []
+#     - ".*<regex>.*"
```

Comment:
these aren't "pure" regex entries actually , this might be a bit confusing. 
Pheraps changing it to `additional_names/services/experience` ? 
Also the `.*` should be removed too right?

Reply:
You are right, better no to confuse the customers!